pom.xml

```diff
@@ -98,6 +98,16 @@
       <groupId>com.fasterxml.jackson.core</groupId>
       <artifactId>jackson-databind</artifactId>
     </dependency>
+    <dependency>
+      <groupId>io.dropwizard</groupId>
+      <artifactId>dropwizard-testing</artifactId>
```

Comment:
Yes because we're using these in the `src/main` code rather than the `src/test` code.